SEED-platform/seed
# Add property measure class method defaults

#### Any background context you want to provide?
`PropertyMeasure`s are imported via an AT submission sync. The `PropertyMeasure` model has 3 choice fields (`category_affected`, `implementation_status`, and `application_scale`), each with a default choice. 

Modify the `PropertyMeasure` model's class methods to handle edge cases where integer choice fields returned None. Previously, the integer fields did not allow null values, and default choices were only applied when the field was entirely omitted, not when `None` was returned for edge cases.
```
# null error
PropertyMeasure.objects.create(category_affected=None) 
# Default scenario
PropertyMeasure.objects.create()
```

The Berkeley data error was related to a single submission (Custom ID 1: 3100). One of the first measures brought in had an unrecognized `category_affected` ("OnSite..." vs "On-Site...") leading to a `None` value being assigned and triggered a non-null constraint. This caused the loop to exit early and prevented any other property updates.

```
seed_postgres  | ERROR:  null value in column "category_affected" violates not-null constraint
```
#### What's this PR do?
- Updates the `PropertyMeasure` class methods to return defaults instead of `None`
- Updates tests and removes developer logs

#### How should this be manually tested?
- Create a 2020 cycle and upload "Berkeley 2023 Benchmarks All Fields.csv". In the mapping step, change "Berkeley Building ID" to "Custom ID 1".
- In Org Settings, set AT credentials for a Berkeley account.
- Trigger AT workflow from: Org Settings, Data Import, Inventory List. Confirm that 4 properties have been updated.

#### What are the relevant tickets?

#### Screenshots (if appropriate)
